src/main/java/Duke.java

```diff
@@ -1,10 +1,119 @@
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
```

Comment:
Since the Duke logo is unlikely to change while in use, would it perhaps be better to declare it as a constant instead?